docs/source/api/v2/asns.rst

```diff
@@ -66,11 +66,11 @@ Request Structure
 
 Response Structure
 ------------------
-:asn:          An :abbr:`ASN (Autonomous System Number)` as specified by IANA for identifying a service provider
-:cachegroup:   A string that is the :ref:`cache-group-name` of the :term:`Cache Group` that is associated with this :abbr:`ASN (Autonomous System Number)`
-:cachegroupId: An integer that is the :ref:`cache-group-id` of the :term:`Cache Group` that is associated with this :abbr:`ASN (Autonomous System Number)`
-:id:           An integral, unique identifier for this association between an :abbr:`ASN (Autonomous System Number)` and a :term:`Cache Group`
-:lastUpdated:  The time and date this server entry was last updated in an ISO-like format
+:asn:		An :abbr:`ASN (Autonomous System Number)` as specified by IANA for identifying a service provider
+:cachegroup:	A string that is the :ref:`cache-group-name` of the :term:`Cache Group` that is associated with this :abbr:`ASN (Autonomous System Number)`
+:cachegroupId:	An integer that is the :ref:`cache-group-id` of the :term:`Cache Group` that is associated with this :abbr:`ASN (Autonomous System Number)`
+:id:		An integral, unique identifier for this association between an :abbr:`ASN (Autonomous System Number)` and a :term:`Cache Group`
+:lastUpdated:	The time and date this server entry was last updated in an ISO-like format
```

Comment:
These looked to all be aligned, why de-align them?

Reply:
Aligned fields with spaces in c2b73f08e9.